tests/test_drive_controller.py

```diff
@@ -120,10 +120,10 @@ def test_motor_rpm_calculations_based_on_speeds(self):
 
         test_values = [
             [0, 0, 0, 0, 0],
-            [95.90, 114.00, 1, 1, 0],
-            [104.58, 114.00, 1, 1, 1],
-            [70.70, 84.10, 0.3, 0.3, 0],
-            [103.62, 114.0, 0.3, 0.3, 0.8],
+            [96.39, 114.00, 1, 1, 0],
```

Comment:
To repeat myself - why?

Reply:
I updated these since these calculations were affected by the change in `DriveController._wheel_separation`